scripts/search/settings.js

```diff
@@ -0,0 +1,84 @@
+// Algolia index settings.
+//
+// Changes to this file should be made with extreme care.
+module.exports = {
+
+    // Attributes for faceting are the properties to use for top-level filtering of results -- for
+    // example, whether a given record is returned for a "Docs" query, "Registry" query, and so on.
+    // https://www.algolia.com/doc/api-reference/api-parameters/attributesForFaceting/
+    getAttributesForFaceting() {
+        return [
+            "searchable(section)",
+        ];
+    },
+
+    // Synonyms allow us to surface content that may not directly match users' queries.
+    // https://www.algolia.com/doc/api-reference/api-methods/save-synonyms/#save-synonyms
+    getSynonyms() {
+        return [
+            ["azuread", "azure ad", "azure active directory"],
+            ["rbac", "role based access control"],
+            ["org", "organization"],
+            ["pulumi service", "pulumi console", "pulumi cloud"],
+            [".NET", "dotnet"],
+            ["C#", "csharp"],
+            ["F#", "fsharp"],
+            ["cfn", "cloudformation"],
+            ["tf", "terraform"],
+            ["stack reference", "stackreference"],
+            ["openid connect", "oidc"],
+            ["crosswalk", "awsx"],
+            ["crossguard", "pac", "policy as code"],
+            ["aws", "aws classic"],
+            ["azure", "azure native"],
+            ["how-to guide", "how to", "example"],
```

Comment:
should we include a synonym without the dash in `how-to guide` as well?

Reply:
I actually meant this to be "how to guide", so will add. Thanks for spotting!